addons/test_mail/static/tests/tracking_value_tests.js

```diff
@@ -82,11 +86,14 @@ QUnit.test("rendering of tracked field of type float: from 0 to non-0", async fu
     const pyEnv = await startServer();
     const mailTestTrackAllId1 = pyEnv["mail.test.track.all"].create({
         float_field: 0,
+        float_field_with_digits: 0,
     });
     await this.start({ res_id: mailTestTrackAllId1 });
-    await insertText("div[name=float_field] input", "1", { replace: true });
+    await insertText("div[name=float_field] input", "1.01", { replace: true });
+    await insertText("div[name=float_field_with_digits] input", "1.0001", { replace: true });
     await click(".o_form_button_save");
-    await contains(".o-mail-Message-tracking", { text: "0.001.00(Float)" });
+    await contains(".o-mail-Message-tracking", { text: "0.001.01(Float)" });
+    await contains(".o-mail-Message-tracking", { text: "0.000000001.00010000(Float)" });
```

Comment:
Isn't there a way to differentiate both trackings? As it stands we don't have a clear understanding of which field uses which precision.